src/quorum/testdata/joint_commit.txt

```diff
@@ -0,0 +1,481 @@
+# No difference between a simple majority quorum and a simple majority quorum
+# joint with an empty majority quorum. (This is asserted for all datadriven tests
+# by the framework, so we don't dwell on it more).
+#
+# Note that by specifying cfgj explicitly we tell the test harness to treat the
+# input as a joint quorum and not a majority quorum. If we didn't specify
+# cfgj=zero the test would pass just the same, but it wouldn't be exercising the
+# joint quorum path.
+committed cfg=(1,2,3) cfgj=zero idx=(100,101,99)
+----
+       idx
+x>       [0]100 (id=1)
```

Comment:
Is it group id? How about hiding 0 by default? So that the file is the same as etcd.

Reply:
I set the width larger for the space of group id

Reply:
Can they be removed?

Reply:
> Can they be removed?

Which one? space, group id or this describe?

Reply:
Space. Better make the file identical with etcd's.